importer/helpers/dagbuilder.go

```diff
@@ -139,7 +155,11 @@ func (db *DagBuilderHelper) GetNextDataNode() (*UnixfsNode, error) {
 			raw:     true,
 		}, nil
 	} else {
-		blk := NewUnixfsBlock()
+		blk := &UnixfsNode{
```

Comment:
Correct.  It was only used in one place so I just inlined the code.  I removed the function to make sure I cached all possible places where it was called to make sure the correct Cid version was always set.